our-process/policies.md

```diff
@@ -2,133 +2,181 @@
 
 ## Review Process Guidelines
 
-pyOpenSci packages are reviewed for quality, fit, scope,
-documentation and usability. The review process
-is similar to a manuscript review, however it has a stronger
-focus on Python packaging best practices.
+pyOpenSci packages are reviewed for quality, fit, scope, documentation, and
+usability. The review process is similar to a manuscript review, however, it
+has a stronger focus on Python packaging best practices.
 
-Unlike a manuscript review, our peer review process is an ongoing conversation. Once all major issues and questions are addressed, the review editor package will make a decision to accept, hold, or reject the package.
+Unlike a manuscript review, our peer review process is an ongoing conversation.
+Once all major issues and questions are addressed, the review editor will make
+a decision to accept, hold, or reject the package.
 
-Rejections are usually done early in the process, before the review process begins. In rare cases a package may also not be on-boarded into the pyOpenSci ecosystem after review & revision.
+Rejections are usually done early in the process, before the review process
+begins. In rare cases, a package may also not be on-boarded into the pyOpenSci
+ecosystem after review & revision.
 
-It is ultimately the editor’s decision on whether or not to reject the package based on how the reviews are addressed.
+It is ultimately the editor’s decision on whether or not to reject the package
+based on how the reviews are addressed.
 
-## Review communication approach
+## Review Communication Approach
 
-Communication between authors, reviewers and editors takes
-place on GitHub. You can, however choose to contact the editor by email if needed.
+Communication between authors, reviewers, and editors takes
+place on GitHub. You can, however choose to contact the editor by email if
+needed.
 
-When submitting a package, please make sure that your GitHub notification settings are setup to notify you when you receive feedback on the review issue.
+When submitting a package, please make sure that your GitHub notification
+settings are setup to notify you when you receive feedback on the review issue.
 
-## Submitting your package for review in other venues
+## Submitting Your Package for Review in Other Venues
```

Comment:
@InessaPawson thank you for this pr!! Please note that we have been trying to use sentence case for h2 and lower headers. H1 can be title case. 

I see this guide is a mix of both. But maybe this is a good time to start enforcing that policy in this guide. The packaging guide i think uses it. Are you open to editing this PR?

Reply:
Thank you for the feedback, @lwasser ! I’ll keep the header styling guidelines in mind for future contributions.